towhee/dag/operator_repr.py

```diff
@@ -107,6 +109,10 @@ def iter_info(self) -> Dict[str, Any]:
         """
         return self._iter_info
 
+    @property
+    def extra(self):
```

Comment:
what dose the `extra` mean?

Reply:
Some parameters for engine, such as model serving batch size, GPU device and so on